iris_grib/grib_phenom_translation.py

```diff
@@ -316,3 +316,70 @@ def cf_phenom_to_grib2_info(standard_name, long_name=None):
     if standard_name is not None:
         long_name = None
     return _CF_GRIB2_TABLE[(standard_name, long_name)]
+
+
+class GRIBcode(namedtuple('GRIBcode',
```

Comment:
I'm afraid I don't like `GRIBcode` I read that as `GRI Bcode`

Maybe it is best to go back to `GribCode` if you don't like GRIBCode
Wikipedia's article on camel links to an paper that points out a run of uppercase letters can cause problems which I think we're having with GRIBCode. So I don't think we have to strictly treat acronyms as all capital letters.

Reply:
> if you don't like GRIBCode

I read as "GRIBC ode".  It seems to be the opposite problem to your "GRI Bcode" !